pallets/parachain-staking/src/lib.rs

```diff
@@ -2718,17 +2734,18 @@ pub mod pallet {
 
                 let growth_creator =
                     growth_info.added_by.ok_or(Error::<T>::GrowthTxSenderNotFound)?;
-                // create_and_report_summary_offence::<T>(
-                //     &reporter,
-                //     &vec![growth_creator],
-                //     SummaryOffenceType::CreatedInvalidRoot,
-                // );
-
-                // create_and_report_summary_offence::<T>(
-                //     &reporter,
-                //     &vote.ayes,
-                //     SummaryOffenceType::ApprovedInvalidRoot,
-                // );
+                    
+                create_and_report_growth_offence::<T>(
+                    &reporter,
+                    &vec![growth_creator],
+                    GrowthOffenceType::CreatedInvalidGrowth,
```

Comment:
wouldn't this error out?

Reply:
Yes, and it was removed in the next PR. But I have also removed it here.